src/networking.c

```diff
@@ -1061,7 +1091,7 @@ void deferredAfterErrorReply(client *c, list *errors) {
     listRewind(errors,&li);
     while((ln = listNext(&li))) {
         sds err = ln->value;
-        afterErrorReply(c, err, sdslen(err));
+        afterErrorReply(c, err, sdslen(err),0);
```

Comment:
redis is inconsistent with these, but let's at least not mix in the same line.
```suggestion
        afterErrorReply(c, err, sdslen(err), 0);
```